copy-from.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+module.exports = function (txt, options) {
+  return new CopyStreamQuery(txt, options)
+}
+
+const { Writable } = require('stream')
+const code = require('./message-formats')
+
+class CopyStreamQuery extends Writable {
+  constructor(text, options) {
+    super(options)
+    this.text = text
+    this.rowCount = 0
+    this._gotCopyInResponse = false
+    this.chunks = []
+    this.cb = null
+    this.cork()
+  }
+
+  submit(connection) {
+    this.connection = connection
+    connection.query(this.text)
+  }
+
+  _write(chunk, enc, cb) {
+    this.chunks.push({ chunk: chunk, encoding: enc })
+    if (this._gotCopyInResponse) {
+      return this.flush(cb)
+    }
+    this.cb = cb
+  }
+
+  _writev(chunks, cb) {
+    Array.prototype.push.apply(this.chunks, chunks)
```

Comment:
Minor: would `this.chunks.push(...chunks)` work?

Reply:
that's better and works in v8